### PR TITLE
Fix the update story and voting error

### PR DESCRIPTION
**Pull request by an-ju**
This PR should complete the migration to the new API gem and db, fixing Issue #3 .

### CODE REVIEW

**Comment by an-ju** [comment]
This PR also addresses Issue #1 and #2 .

File: app/models/story.rb [hunk 1]
```diff
@@ -2,8 +2,11 @@ class Story
   class << self
 
     def update(params)
-      story = User.client.project(params[:id].to_i).story(params[:story_id].to_i)
-      story.update(params[:story])
+      story = TrackerApi::Resources::Story.new( client:     params[:client],
+                                                project_id: params[:project_id],
+                                                id:         params[:story_id])
+      story.attributes = params[:story]
+      story.save
```
Comment (adnanhemani) [inline app/models/story.rb]: This got recently changed in my code - will mix this in

File: app/models/user.rb [hunk 2]
```diff
@@ -20,14 +20,12 @@ def authenticate(params)
...
-        puts err.response
-        return
-      end
-
+      api_response = RestClient::Request.execute(
+          method: :get,
+          url: "https://www.pivotaltracker.com/services/v5/me",
+          user: params[:username],
+          password: params[:password]) # HTTP errors are handled by application_controller.rb
+      token = JSON.parse(api_response)["api_token"]
```
Comment (adnanhemani) [inline app/models/user.rb]: Better to be defensive here and use begin/rescue - but your formatting is much better here

**Review by adnanhemani — COMMENTED** [review]
Going to merge these changes with the latest push I made and then will push

File: app/models/user.rb (hunk 2 above)
Comment (adnanhemani) [reply app/models/user.rb]: Oh sorry - didn't see your comment, good point!

Comment (adnanhemani) [reply app/models/user.rb]: Actually, jsut tested it out, for some reason application_controller.rb doesn't handle if the user/pass is incorrect. So I'll keep the code as-is. The log is much better to look at anyways

**Pull request merged**